src/transformers/configuration_utils.py

```diff
@@ -249,6 +247,11 @@ def __init__(self, **kwargs):
 
         # TPU arguments
         self.xla_device = kwargs.pop("xla_device", None)
+        if kwargs.pop("xla_device", None) is not None:
```

Comment:
The `xla_device` has already been popped, no? Do you mean to use `self.xla_device` instead?

Reply:
Forgot to remove the previous line.